core/src/dbs/iterator.rs

```diff
@@ -19,19 +19,18 @@ use crate::sql::thing::Thing;
 use crate::sql::value::Value;
 use async_recursion::async_recursion;
 use std::mem;
-use std::sync::Arc;
 
 #[derive(Clone)]
 pub(crate) enum Iterable {
 	Value(Value),
-	Table(Arc<Table>),
+	Table(Table),
```

Comment:
A trade-off between Arc<Table> (64 bits pointer + reference counting) and copying the table name